eoscostarica/gGoods
# Create nft template

### Create nft template

### What does this PR do?

- Resolve #41 

### Steps to test

1. first step
2. navigate to http://localhost:3000/create-template
3. enter a category name
4. enter an asset name
5. click on confirm button

#### CheckList
